client/src/main/scala/com.gu.contentapi.client/model/MetaResult.scala

```diff
@@ -3,43 +3,43 @@ package com.gu.contentapi.client
 import com.gu.contentapi.client.model.v1._
 import com.gu.contentatom.thrift.Atom
 
-private[client] trait MetaResult[A] {
-  type R
-  def getCurrentPage: A => Int
-  def getTotalPages: A => Int
-  def isLastPage: A => Boolean = a => getCurrentPage(a) == getTotalPages(a)
-  def getResults: A => Seq[R]
-  def getId: R => String
+trait MetaResult[Response] {
+  type Result
+  def getResults: Response => Seq[Result]
+  def getId: Result => String
+  def getCurrentPage: Response => Int
+  def getTotalPages: Response => Int
+  def isLastPage: Response => Boolean = a => getCurrentPage(a) == getTotalPages(a)
 }
 
 object MetaResult {
 
   implicit val searchResponse = new MetaResult[SearchResponse] {
-    type R = Content
+    type Result = Content
     def getCurrentPage = _.currentPage
     def getTotalPages = _.pages
     def getResults = _.results
     def getId = _.id
   }
 
   implicit val itemResponse = new MetaResult[ItemResponse] {
-    type R = Content
-    def getCurrentPage = _.currentPage.getOrElse(1)
-    def getTotalPages = _.pages.getOrElse(1)
+    type Result = Content
+    def getCurrentPage = _.currentPage.getOrElse(0)
+    def getTotalPages = _.pages.getOrElse(0)
```

Comment:
It would be useful to see a comment here describing why we should return `0`, not `1`.